doc/rules/alias/backtick_to_shell_exec.rst

```diff
@@ -0,0 +1,27 @@
+===============================
+Rule ``backtick_to_shell_exec``
+===============================
+
+Converts backtick operators to ``shell_exec`` calls.
+
+Description
+-----------
+
+Conversion is done only when it is non risky, so when special chars like
+single-quotes, double-quotes and backticks are not used inside the command.
+
+Examples
+--------
+
+Example #1.
```

Comment:
I would make this a header, so it is linkable and have better layout, as well as remove the full stop

```suggestion
Example #1
~~~~~~~~
```

Reply:
Done.